patch/no-sgx-flc/README.md

```diff
@@ -1,6 +1,7 @@
 There are still non-trivial number of systems without FLC support.
 
 # Prerequisite
+- Apply the patch `0001-SGX-DCAP-Linux-Driver-Support-SGX1-machine-even-without-FLC-s.patch` to [SGX DCAP Linux driver](https://github.com/intel/SGXDataCenterAttestationPrimitives).
```

Comment:
You need to highlight the application of either dcap version or in-tree version used.